setup.py

```diff
@@ -18,13 +19,23 @@
 history = open('CHANGES.rst').read()
 
 tests_require = [
+    'check-manifest>=0.42',
+    'coverage>=5.3,<6',
+    'docker-services-cli>=0.4.0',
+    'msgpack-python>=0.5.6',
+    'pytest-cov>=3.0.0',
+    'pytest-flask>=1.2.0',
+    'pytest-isort>=3.0.0',
     'pytest-mock>=2.0.0',
-    'pytest-invenio>=1.4.0',
```

Comment:
I've removed pytest-invenio, since it adds a lot of Invenio module dependencies that's not needed (just makes pip resolver takes longer)